Doc/library/functions.rst

```diff
@@ -1388,6 +1388,17 @@ are always available.  They are listed here in alphabetical order.
    object allows it.  For example, ``setattr(x, 'foobar', 123)`` is equivalent to
    ``x.foobar = 123``.
 
+.. note::
+
+   setattr() attempts to update the object with the given attr/value pair.
+   Whether this succeeds and what its affect is is determined by the target object.
```

Comment:
I would suggest changing `what its affect is` to `the affect it has` so that `is is` doesn't trigger future (incorrect) doc fixes.

Reply:
Actually, I think `affect` should be `effect`.